### PR TITLE
Add helpers.py file with some helper functions to ensure DRY principle

### PR DESCRIPTION
**Pull request by nahidsaikat**
I was going through the code base and found that there are some duplicate codes in live_unread_notification_list and live_all_notification_list views which is a violation of DRY principle. I thought some helper functions can remove the duplicate codes. So I have written down the helper functions and removed duplicate codes.

### CODE REVIEW

**Comment by timthelion** [comment]
My PR from a week ago https://github.com/django-notifications/django-notifications/pull/278/files does basically the same thing.

File: notifications/views.py [hunk 1]
```diff
@@ -5,18 +5,20 @@
 
 from django import get_version
 from django.contrib.auth.decorators import login_required
-from django.forms import model_to_dict
 from django.shortcuts import get_object_or_404, redirect
 from django.utils.decorators import method_decorator
 from django.views.decorators.cache import never_cache
 from django.views.generic import ListView
 from notifications import settings
+from notifications.models import Notification
```
Comment (AlvaroLQueiroz) [inline notifications/views.py:13]:
```suggestion
```

**Comment by AlvaroLQueiroz** [comment]
@nahidsaikat can you rebase your branch, please?

**Comment by AlvaroLQueiroz** [comment]
Done in another PR

**Pull request closed without merging**